experiments/dataSources/airtable/README.md

```diff
@@ -81,20 +79,98 @@ If you've set the data source URL correctly, clicking on **Fields** will show yo
+Configuring delete is easiest. 
+
+- **Type**: "DELETE"
+- **Data Source URL**: ``<baseId>/<ObjectName>/{{id}}``
+- **Data Source HTTP Verb**: ``DELETE``
+
+This Data Source URL takes the ID of one record and passes it to Airtable to delete.  You can use the "Mark row(s) for deletion" and "Save" actions to trigger this method.  
+
+### Insert
+Lets create a new record. 
+
+- **Type**: "INSERT"
+- **Data Source URL**: ``<baseId>/<ObjectName>``
+- **Data Source HTTP Verb**: ``POST``
+- **Send new field values**: ``As templated request body``
+- **Method succeeds if...**: ``Request Succeeds``
+- **Response handling**: ``Created Record will be returned``
+
+You configure exactly how the new fields are passed to Airtable in the Custom Request template. 
+Below is the template for creating a record with all the fields in the applicants table. 
+
+```json
+{"fields": {
+        "Email Address": "{{fields.Email Address}}",
+        "Name": "{{fields.Name}}",
+        "Applying for": [
+          "{{fields.Applying for}}"
+        ],
+        "Onsite Interview Date": "{{fields.Onsite Interview Date}}",
+        "Onsite Interview Notes": "{{fields.Onsite Interview Notes}}",
+        "Onsite Interview Score": "{{fields.Onsite Interview Score}}",
+        "Onsite Interviewer": [
+          "{{fields.Onsite Interviewer}}"
+            ],
+        "Phone": "{{fields.Phone}}",
+        "Phone Screen Date": "{{fields.Phone Screen Date}}",
+        "Phone Screen Notes": "{{fields.Phone Screen Notes}}",
+        "Phone Screen Score": "{{fields.Phone Screen Score}}",
+        "Phone Screen Interviewer": [
+          "{{fields.Phone Screen Interviewer}}"
+        ],
+        "Stage": "{{fields.Stage}}"
+        }
+}
+```
+### Update
+Lets update an existing record.  
+
+- **Type**: "UPDATE"
+- **Data Source URL**: ``<baseId>/<ObjectName>/{{id}}``
+- **Data Source HTTP Verb**: ``PATCH``
+- **Send new field values**: ``As templated request body``
+- **Method succeeds if...**: ``Request Succeeds``
+
+The same request body template used for ``Insert`` can be used for ``update``
+
+
 ## Sample page
 
 The included sample page is based off of Airtable's default Applicant Tracking base. It has one model for the Applicants sheet, aka object, and it lists all of the default columns. To see it in action, do the following:
 
 1. Create the data source as outlined above
-2. Upload the XML to your Skuid site as a **v1** page.
-3. **You will see an error!** This is because you need to update the base ID for the model. Replace the `<Your_Base_Id>` string within the model's data source URL with **your base ID.**
-4. Save and preview.
+2. Upload the XML to your Skuid site as a **v2** page.
+3. **You will see an error!** This is because you need to update the base ID for the model. Replace the `<Your_Base_Id>` string within all of the model's data source URLs with **your base ID.**  (There are 6 instances where you will have to replace this string - multiselect is your friend)
```

Comment:
```suggestion
3. **You will see an error!** This is because you need to update the base ID for the model. Replace the `<Your_Base_Id>` string within all of the model's data source URLs with **your base ID.**  (There are 6 instances where you will have to replace this string - multiselect while editing XML is your friend)
```

Reply:
I didn't know what you meant at first, but this is it right?

Reply:
You can multiselect in the XML editor...  The key bindings will change with Monaco,  but its wonderful...

Reply:
ahhh editing my suggestion then

Reply:
Updated.

Reply:
Actually - Im just gonna take out that comment.